clintfidel/WEconnect
# #156593926 Create and connect register business component

#### What does this PR do?
This PR enables users register new business.

#### Description of Task to be completed?
This PR ensures that on sucessful sign-up or login,visiting the required route, users are able to register a business succesfully.

#### How should this be manually tested?
- Signup or login to the application
- Navigate to the register business page
- Then you should be able to register a business successfully

#### What are the relevant pivotal tracker stories?
[156593926](https://www.pivotaltracker.com/story/show/156593926)

#### Any background context you want to add?
N/A

#### Screenshots
![image](https://user-images.githubusercontent.com/26988297/38457673-d8e5b176-3a8b-11e8-8eac-25a67ba90593.png)
